docs/AI-for-security/api/chat-complete-api.asciidoc

```diff
@@ -0,0 +1,117 @@
+[[chat-complete-api]]
+=== Chat Complete API
+
+The Chat Complete API lets you create new Elastic AI Assistant conversations, and add messages to existing conversations.
```

Comment:
I will extend the description, because chat/complete API is the different thing. It allows to communicate to the configured LLM and if needed to persist the result as a conversation (new or extend existing).